server/src/routes/root.js

```diff
@@ -1,8 +1,10 @@
-const example = require('./example.route');
-
 const express = require('express');
 const router = express.Router();
 
+const example = require('./example.route');
+const authRoutes = require("./routes/auth");
+
 router.use('/example', example);
+router.use("/api", authRoutes);
```

Comment:
Can you pls add the appropriate route path? As currently due to this the signup will hit if the path is `/api/api`, and that seems pretty unreasonable